crate/content.md

```diff
@@ -1,111 +1,41 @@
-# HOST="crate1.example.com"
-# docker run -d -p 4200:4200 -p 4300:4300 \
-    --name crate1-container \
-    --volume /mnt/data:/data \
-    --ulimit nofile=65535 \
-    --ulimit memlock=9223372036854775807 \
-        crate \
-        crate \
-          -Ccluster.name=crate-cluster \
-          -Cnode.name=crate1 \
-          -Ctransport.publish_port=4300 \
-          -Cnetwork.publish_host="$HOST" \
-          -Cmulticast.enabled=false \
-          -Cdiscovery.zen.ping.unicast.hosts="$HOSTS" \
-          -Cdiscovery.zen.minimum_master_nodes=2
-```
-
-# The crate Docker Image
-
-To form a cluster from scratch, start a few instances of the Crate container as a background daemon:
-
-```console
-# docker run -d crate crate
-```
-
-To access the admin UI, map port 4200 and point your browser to port tcp/4200 of a node of your choice while you start it or look up its IP later:
-
-```console
-# firefox "http://$(docker inspect --format='{{.NetworkSettings.IPAddress}}' $(docker run -d crate crate)):4200/admin"
-```
-
-For production use it's strongly recommended to use only one container per machine. This will give the best possible performance and by mapping the ports from the Docker container to the host it acts like a native installation. Crate's default ports 4200 (HTTP) and 4300 (Transport protocol).
-
-```console
-# docker run -d -p 4200:4200 -p 4300:4300 crate crate
-```
-
-## Attach Persistent Data Directory
-
-Crate stores all important data in */data*. It's advised to mount this directory to avoid writing within the docker image:
-
-```console
-# docker run -d -v <data-dir>:/data crate crate
-```
-
-## Use Custom Crate Configuration
+# What Is CrateDB?
 
-Starting with 0.55.0, Crate does no longer support providing custom configuration files. However it is still possible to mount Crate's configuration into `/crate/config/crate.yml`.
+[CrateDB](github.com/crate/crate) is a distributed SQL database that makes it simple to store and analyze massive amounts of machine data in real-time.
 
-```console
-# docker run -d -v <custom/config/path>/crate.yml:/crate/config/crate.yml crate crate
-```
+Features of CrateDB:
 
-For further configuration options refer to the[Configuration](https://crate.io/docs/stable/configuration.html) section of our documentation.
+	- Standard SQL plus dynamic schemas, queryable objects, geospatial features, time series data, first-class BLOB support, and realtime full-text search.
+	- Horizontally scalable, highly available, and fault tolerant clusters that run very well in virtualized and containerised environments.
+	- Extremely fast distributed query execution.
+	- Auto-partitioning, auto-sharding, and auto-replication.
+	- Self-healing and auto-rebalancing.
+	- CrateDB offers the scalability and flexibility typically associated with a NoSQL database and is designed to run on inexpensive commodity servers and can be deployed and run across any sort of network. From personal computers to multi-region hybrid clouds.
```

Comment:
These tabs are reversed: (should be "hypen<tab>text")
```markdown
-	Standard ...
```
(Likely being treated as a code-block as-is)